devtools/running_packages.rst

```diff
@@ -192,12 +209,9 @@ Installing and running this package, can be done by any of the means presented a
 
 .. code-block:: bash
 
-    $ conan install HelloRun/0.1@user/testing -g virtualrunenv
+    $ conan install . -g virtualrunenv
```

Comment:
In this case it wants to show how to directly install packages, without a local conanfile